tests/Behat/features/element-editor.feature

```diff
@@ -0,0 +1,90 @@
+@javascript
+Feature: View types of elements in a report
+  As a CMS user
+  I want to view a list of elements in the CMS
+  So that I can see which elements I have used on a page
+
+  Background:
+    Given I have a config file "enable-elemental.yml"
+    Given I am logged in with "ADMIN" permissions
+      And I add an extension "DNADesign\\Elemental\\Extensions\\ElementalPageExtension" to the "Page" class
+      And I go to "/dev/build?flush"
+      And a "page" "Blocks Page"
+      And I go to "/admin/pages"
+      And I wait until I see the ".cms-tree" element
+      And I click on "Blocks Page" in the tree
+      And I wait until I see the ".elemental-editor__list" element
+      # Workaround until the FixtureFactory applies ElementalArea relations correctly;
+      # See https://github.com/silverstripe/silverstripe-behat-extension/issues/180
+      And I select "Content" from "elemental-editor_add-new-block-control_select-dropdown"
+      And I click "Add" in the ".elemental-editor__add-new-block-control" element
+      And I fill in "My Sample Block" for "Title"
+      And I fill in "<p>My sample content</p>" for the "HTML" HTML field
+      And I click "Create" in the "#Form_ItemEditForm_action_doSave" element
+      # Remove with 'And I click "Blocks Page" in the ".breadcrumbs-wrapper" element' once the ElementalArea refreshes,
+      # See https://github.com/dnadesign/silverstripe-elemental/issues/320
+      And I go to "/admin/pages/edit/show/6"
+      And I wait until I see the ".element-editor__element" element
+    Then I should see "My Sample Block"
+
+  Scenario: I can see the icon, title, and summary of each element
+    Given I wait until I see the ".element-editor__element" element
+    Then I should see a list of blocks
+      And I should see "My Sample Block" as the title for block 1
+
+  Scenario: I can add elements to the page
+    When I select "Content" from "elemental-editor_add-new-block-control_select-dropdown"
+      And I click "Add" in the ".elemental-editor__add-new-block-control" element
+      And I fill in "Additional Sample Block" for "Title"
+      And I fill in "<p>Additional sample content</p>" for the "HTML" HTML field
+      And I click "Create" in the "#Form_ItemEditForm_action_doSave" element
+    Then I should see an ".message" element
```

Comment:
You could probably remove this and make the next line `Then I should see a "Saved content block" message`